tensorflow/python/training/experimental/mixed_precision_test.py

```diff
@@ -117,7 +117,10 @@ def test_optimizer_errors(self):
     self.assertFalse(config.get_optimizer_experimental_options()
                      .get('auto_mixed_precision', False))
 
-  # TFDML #25510022
+  # DML doesn't have float16 optimizations in grappler yet, so we wouldn't
+  # overflow in this case
+  # TODO #26542950: Enable Tensorcore specific grappler optimizations
+  # TFDML #26542950
```

Comment:
Was this change supposed to disable something new, or just add comments?

Reply:
Just adding comments (I should rename the PR). Now that the cause for the disabled test is known, we can close the old generic bug that was created to investigate the failure and instead link it to the grappler optimization task.